modules/ROOT/pages/database-administration/composite-databases/delete-composite-databases.adoc

```diff
@@ -12,3 +12,36 @@ However, keep in mind that the first command will fail if the target database is
 DROP COMPOSITE DATABASE inventory
 ----
 
+[[composite-databases-delete-with-aliases]]
+== Delete a composite database with constituent database aliases
+
+In general, deleting a composite database that has constituent database aliases will fail due to those database aliases.
+This affects both the local and remote constituent database aliases belonging to the specified composite database.
+
+By appending `CASCADE ALIASES` to the command `DROP COMPOSITE DATABASE`, those constituent database aliases can be dropped together with the composite database.
+This affects both the local and remote constituent database aliases belonging to the specified composite database.
+
+Using `CASCADE ALIASES` requires the `DROP ALIAS` privilege (see xref:authentication-authorization/dbms-administration.adoc#access-control-dbms-administration-alias-management[ALIAS MANAGEMENT privileges]).
+
+Given composite database and database alias:
+[source, cypher]
+----
+CREATE COMPOSITE DATABASE movies
+CREATE ALIAS movies.sweden FOR DATABASE `swedish-movies`
+----
+
+Then both the constituent database alias `sweden` and the composite database `movies` will be dropped with:
+[source, cypher]
+----
+DROP COMPOSITE DATABASE movies CASCADE ALIASES
+----
+
+This behaviour would be the same for the more general one `DROP DATABASE name` as well when using it to drop composite databases.
+
+The option `RESTRICT` explicitly requests the default behavior of the command.
+
+[NOTE]
+====
+The `CASCADE ALIASES` option will drop the same aliases that would be given in the error message from trying to drop the database using the `RESTRICT` mode instead.
```

Comment:
Do we need this note? It seems like it could easily get broken by changing the error message and it kinda restates the above.

Reply:
I'm not entirely convinced by the comment, but if we want to address it, I don't think this is a good way because running the command to get the error message will actually drop the database if there are no aliases.

Perhaps something like: 

The aliases that will be dropped can be shown with `SHOW DATABASE db YIELD name, constituents` . And the same for aliases for the other page.

Reply:
I mean you were kinda after trying to drop the database regardless 🤷 but I'll update to something along those lines tomorrow 👍